### ISSUE TITLE
Plan for pulling this back to Private

### ISSUE THREAD

**Issue by dwiserCP** [colpal/cp-dbt-standard-package#1]
I'd like to have the team figure out how to make this repo back to Private.

**Comment by KevinDukelow5** [comment]
Agreed on the outcome here. We would want a way to distribute our internally built dbt packages.

We have several environments which are important to support:

GitHub actions
Local DBT cli
DBT cloud

I'm not clear what the best way to achieve this is but I do see that the official dbt hub site is OS: https://github.com/fishtown-analytics/hub.getdbt.com/tree/master
